templates/definition/charger/openwb-2.0.yaml

```
@@ -6,8 +6,16 @@ products:
 capabilities: ["1p3p", "mA"]
 requirements:
   description:
-    de: Erfordert Software 2.x
-    en: Requires software 2.x
+    de: |
+      Erfordert [`Software 2.x`](https://github.com/openWB/core).
+      Folgende Änderungen sind unter dem `Einstellungen`-Reiter erforderlich:
+      * Steuerungsmodus: `secondary`
+      * Steuerung über Modbus als secondary: `An`
```

```suggestion

      * Steuerung über Modbus als secondary: `An`
```

There needs to be a blank line above the listing for Markdown to format this correctly.

---

Hi, I based the syntax on [this file](https://github.com/evcc-io/evcc/blob/72fca28c093cfe0660122c755af49b5af59d3eb8/templates/definition/charger/ocpp-pulsarplus.yaml), which looks correctly rendered and has no blank line.
Does this mean this file has the same error or are these two files not comparable?

---

See here: https://docs.evcc.io/docs/devices/chargers#wallbox-pulsar-plus-pulsar-max-commander-2-copper-sb

---

Ok, it seems to be a bit complicated. Some Markdown implementations require a blank line, some don't. 

> Since it is well established Markdown practice to allow lists to interrupt paragraphs inside list items, the principle of uniformity requires us to allow this outside list items as well. (reStructuredText takes a different approach, requiring blank lines before lists even inside other list items.)

from https://spec.commonmark.org/0.26/#list

The library Docosaurus doesn't seem to require the newline. For Config UI I haven't picked a library yet.

I'd say lets use the blank line here to be universally compatible. I may have to go through the existing templates and unify that.

---

Let's do it. I can go through the templates for you as long as you run `make docs` for me. :D

---

The blank line has to be between the text-block and the first list item. Not inside the list items.

Regarding `make docs`. It's not necessary any more to run the command as part of a template change. The generated output is not under version control any more. It's built by the pipeline.

---

Done. 👍